### DISCUSSION TITLE
Is backing up the db enough?

### DISCUSSION THREAD

**Discussion by eulenmongol** [requarks/wiki#5908] [Help / Questions] [answered]
What contains the db? Is it safe to just backup the db to another location or do I have to enable "local Backup" and move this backup files also to another location?

- Windows Server 2019
- Wiki.js V2.X.XXX

**Comment by NGPixel** [comment, marked as answer]
The db contains everything so yes that is sufficient. Local backup is simply an extra layer that is for pages only.

**Reply by eulenmongol** [reply]
great! Thank you.
